babel-watch.js

```diff
@@ -156,12 +157,13 @@ const watcher = chokidar.watch(program.watch, {
 let watcherInitialized = (program.watch.length === 0);
 debugInit('Initializing babel-watch with options: %j', program.opts());
 
-process.on('SIGINT', function() {
-  debugInit('SIGINT caught, closing.');
+
+onExit(function(code, signal) {
+  debugInit(`${signal} caught, closing.`);
```

Comment:
Wonder if it should print a message if exited via process.exit. Might add signal
```
signal || `code ${code}`
```